### ISSUE TITLE
Creation of the bin files given path to ply file

### ISSUE THREAD

**Issue by tamireiny** [IntelVCL/FastGlobalRegistration#11]
Hello, 

Can you provide source code for creating the FEST features (the bin files) given a point cloud (ply file)?
Seems to me that It can help anyone trying to use FGR.

Thanks a lot, 
Tamir

**Comment by syncle** [comment]
Hi Tamir,

I think you meant to say FPFH feature. The quick answer we can give you is using [Open3D](http://www.open3d.org). Please check it from [this link](http://open3d.org/docs/tutorial/Advanced/global_registration.html#extract-geometric-feature) to extract FPFH features from PCL file. FYI, [fast global registration will be part of Open3D](https://github.com/IntelVCL/Open3D/issues/142).

**Comment by tamireiny** [comment]
Hi Jaesik, 

Thanks for the quick answer before. 

I wrote a [python script](https://www.dropbox.com/s/79xujugyd6ti4pr/main_shared.py?dl=0) that creates the FPFH feautre using Open3D, this part seems to go well. I am writing the bin file according to the instructions mentioned, and when using read_features (matlab version) it seems that nothing is returned. 

Also, it seems that I need to set several parameters: 
voxel_down_sample: factor
estimate_normals: radius, max_nn
compute_fpfh_feature:radius, max_nn

I saw in each of FGR synthetic folders (i.e airwise_no_noise_01_rot_05) the file alignment.config.
I was not able to match between these files values to the parameters I mentioned above, 

Can you please take a look at the python script link?
Many thanks, 
Tamir

**Comment by syncle** [comment]
I think the link for the python script is broken.

Here is a guideline for parameter tuning. From #7, 
> We recommend 1:2:5 for radius of downsampling voxel size, normal estimation, and feature estimation.

When we generated synthetic dataset, we have normalized the scale of the geometry, so it is not absolute scale for radius. Therefore, it is not metric scale. What we did is we normalize the scale of the geometries, and use the relative scale that meets 1:2:5 rule.
